src/partitioner/local/Partition.cpp

```diff
@@ -0,0 +1,139 @@
+#include "Partition.h"
+#include <iostream>
```

Comment:
I think we should rename the Partition class to "JasmineGraphIncrementalStore".

Reply:
Noted, I have fixed this in following PR https://github.com/miyurud/jasminegraph/pull/52